eth/protocols/eth/broadcast.go

```diff
@@ -142,18 +142,18 @@ func (p *Peer) announceTransactions() {
 		if done == nil && len(queue) > 0 {
 			// Pile transaction hashes until we reach our allowed network limit
 			var (
-				hashes  []common.Hash
+				i       int
```

Comment:
since `i` is no longer only used as a loop index, could you give it a more explicit name?